From 31d840488b78ad9a42e9edbcfc4b9ea11c5ea470 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Mon, 23 Sep 2024 12:55:56 +0300
Subject: [PATCH] [FSW-7256] fs1: transaction initiators list

Added list of initiators that we want to unblock
accessing few areas which required during rom
recovery.

Change-Id: I6799d9e46a3bce35e8f23546e8b59f06fdb4f385
---

diff --git a/arch/fs1/arc/dev_security.h b/arch/fs1/arc/dev_security.h
index 59d0203..700f579 100644
--- a/arch/fs1/arc/dev_security.h
+++ b/arch/fs1/arc/dev_security.h
@@ -64,6 +64,13 @@
 	uint32_t data_max;
 };
 
+/* List of transaction initiators */
+enum security_initiators {
+	SECURITY_INITIATOR_PCIE,
+	SECURITY_INITIATOR_I2C_S_MSTR,
+	SECURITY_INITIATOR_MAX,
+};
+
 void security_load_revoke_bits(void);
 int security_efuse_image_validation_en(void);
 void boot_rom_eom_overrides(void);
